docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,39 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
+meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
+navigation area. After clicking policies, you can view all policies in your meshStack instance. Since this is a BETA
+version we only support tags for policies between meshObjects.
+
+## Create Policies
+
+Policy creation is only possible for a user who has a partner admin role. There is a 'Create Policy' button top
+right on the policies screen. After clicking on it, there is a redirection to the screen of "Create Policy" where all
+details need to be filled out. To create the policy, you need to add the policy name in the form. Then you need to add
+both the meshObjects from the dropdown menu. For ease of use, we already restricted the object combinations which
+do not make sense. You can choose only possible object combinations. Once you select both objects, the section for
+adding rules will appear where you can select the tag of selected meshObject. It is a global rule which will be applied
+everywhere. The created policy has been applied to every configuration which has selected meshObjects.
+
+### Working explanation
+
+You can choose a possible tag for your organization for a particular meshObject. E.g. if you choose meshProject and
+meshLandingZone, and rule applied on environment tag with operator 'in' (which is the only operator we support now).
+Operator 'in' works like an intersection between the objects. Then you can assign a landing zone with prod environment
```

Comment:
inconsistent use of object sometimes its meshObject sometimes object

Reply:
fixed